### ISSUE TITLE
Add PHP 7.3 to .travis.yml; Remove 7.0

### ISSUE THREAD

**Issue by benlk** [INN/Largo#1579]
- [ ] Add PHP 7.3 to the Travis CI build Matrix for the following WordPress versions:
    - [ ] 5.0
    - [ ] 4.9.9 if it's released, if it contains PHP 7.3 support
- [ ] Remove PHP 7.0
- [ ] get the Gutenberg/Classic Editor tests to work, from https://github.com/INN/largo/issues/1514

**Comment by benlk** [comment]
This is blocked by Travis not yet supporting PHP 7.3 yet; their [latest build environment](https://changelog.travis-ci.com/ubuntu-xenial-16-04-build-environment-is-here!-79690) doesn't have PHP 7.3 yet. It only has 5.6, 7.1, and 7.2.

**Comment by benlk** [comment]
If we want PHP 7.3, we will need to switch to the Xenial build environment: https://docs.travis-ci.com/user/reference/bionic/#php-support

This will require dropping PHP 5.6 support, which we should be okay with doing because PHP 5.6 went end-of-life at the end of 2018: https://www.php.net/eol.php

**Comment by benlk** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Add PHP 7.3 to travis tests](https://trello.com/c/IijjLeoe/59-add-php-73-to-travis-tests)

**Comment by benlk** [comment]
Closing in favor of https://github.com/INN/largo/issues/1801, which is more up-to-date.
